src/NodaTime.Demo/OffsetDemo.cs

```diff
@@ -62,6 +62,58 @@ public void ConstructionFromTimeSpan()
             Assert.AreEqual(5400, offset.Seconds);
         }
 
+        [Test]
+        public void Add()
+        {
+            var lefHandOffset = Offset.FromHours(5);
+            var rightHandOffset = Offset.FromHours(6);
+
+            var result = Snippet.For(Offset.Add(lefHandOffset, rightHandOffset));
+
+            Assert.AreEqual(result, lefHandOffset + rightHandOffset);
+        }
+
+        [Test]
+        public void Substract()
+        {
+            var lefHandOffset = Offset.FromHours(7);
+            var rightHandOffset = Offset.FromHours(5);
+
+            var result = Snippet.For(Offset.Subtract(lefHandOffset, rightHandOffset));
+
+            Assert.AreEqual(result, lefHandOffset - rightHandOffset);
+        }
+
+        [Test]
+        public void CompareTo()
+        {
+            var smallerOffset = Offset.FromHours(3);
+            var largerOffset = Offset.FromHours(5);
+            var equalOffset = Offset.FromHours(5);
+
+            var lessThan = smallerOffset.CompareTo(largerOffset);
+            var equalTo = largerOffset.CompareTo(equalOffset);
+            var largerThan = largerOffset.CompareTo(smallerOffset);
+
+            Assert.Less(lessThan, 0);
+            Assert.AreEqual(equalTo, 0);
+            Assert.Greater(largerThan, 0);
+        }
+
+        [Test]
+        public void Equals()
+        {
+            var equalOffset1 = Offset.FromHoursAndMinutes(1, 30);
+            var equalOffset2 = Offset.FromHoursAndMinutes(1, 30);
+            var inequalOffset = Offset.FromHours(2);
+
+            var isEqual = equalOffset1.Equals(equalOffset2);
+            var isInequal = equalOffset1.Equals(inequalOffset);
+
+            Assert.IsTrue(isEqual);
```

Comment:
We don't support IsTrue and IsFalse, but we do support the True and False methods.
I can add support for those if you want, but it's probably simplest to just use Assert.True/False for now.

See https://github.com/nodatime/nodatime.org/blob/master/build/SnippetExtractor/SnippetRewriter.cs#L101 for the transformations we perform.